### ISSUE TITLE
QChip remove icon not visible when text is long

### ISSUE THREAD

**Issue by ajcastro** [quasarframework/quasar#6154]
#### Version
Pkg quasar........ v1.1.4
Pkg @quasar/app... v1.1.2
(I cancelled upgrading to latest version because there are errors happened in ui/css)

#### Description
The remove icon is not visible when the text is long. 
![image](https://user-images.githubusercontent.com/4918318/72696422-13f21280-3b77-11ea-8ebb-1a639cabdfa3.png)

I solved it using the following css.
```css
.q-chip.ellipsis .q-chip__content {
  text-overflow: ellipsis;
  white-space: nowrap;
  overflow: hidden;
}
```
then I add `ellipsis` class in q-chip:
```html
<q-chip class="ellipsis">
</q-chip>
```
![image](https://user-images.githubusercontent.com/4918318/72696491-53206380-3b77-11ea-9873-0ef2944715b5.png)

**Comment by pdanpdan** [comment]
Please make a codepen.

**Comment by rstoenescu** [comment]
No codepen necessary. Fixing it.

**Comment by rstoenescu** [comment]
Fix will be available in "quasar" v1.8.0.
